### PR TITLE
Csv implementation

### PR DESCRIPTION
**Pull request by bladekey88**
Re-implemented CSV export functionality. 

There is a lot of duplicate code between the JSON and CSV functionalities, that can probably be pulled out into a general export class/interface

### CODE REVIEW

File: ENVParser/Utils/CsvExporter.cs
Comment (DarkPsydeOfTheMoon) [inline ENVParser/Utils/CsvExporter.cs:57]: This definitely looks similar to some stuff in the JSON code. Is this sort of thing what you mean when you mention refactoring later to consolidate pieces of the code?

**Review by DarkPsydeOfTheMoon — APPROVED** [review]
LGTM -- I agree that a code-consolidating refactor between this and the JSON code can wait until a future PR after this is all merged.

File: ENVParser/Utils/CsvExporter.cs
Comment (bladekey88) [reply ENVParser/Utils/CsvExporter.cs:57]: Yes, along with the actual way I've implemented the CSV export.

**Pull request merged**
